moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -187,7 +191,7 @@ const ProjectActivityLog = () => {
   return (
     <ApolloErrorHandler error={error || lookupError}>
       <CardContent>
-        <h2 style={{ padding: "0rem 0 2rem 0" }}>Activity feed</h2>
+        <h2 className={classes.projectPageHeader} style={{ padding: "0rem 0 2rem 0" }}>Activity feed</h2>
```

Comment:
do you think we could fold the padding style into the `projectPageHeader` style?

Reply:
I think we absolutely could and should - thank you for suggesting that. I'll amend the PR here momentarily.